angularjs-portal-home/src/main/webapp/partials/static-content-card.html

```diff
@@ -8,8 +7,6 @@
      <i title="Remove" class="fa fa-times portlet-options" ng-click="layoutCtrl.removePortlet(portlet.nodeId, portlet.title)"></i>
   </div>
   <div class="portlet-content">
-    {{:: portlet.description | truncate:160}}
+    <div ng-bind-html="portlet.pithyStaticContent"></div>
```

Comment:
Will re-work the changeset to introduce `pithy-content-card` rather than modifying `static-content-card`.